Framework/PythonInterface/mantid/utils/path.py

```diff
@@ -0,0 +1,44 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+# 3rd party
+from mantid import config, FileFinder
+
+# standard
+from pathlib import Path
+from typing import Optional, Union
+
+
+def run_file(run_number: Union[str, int],
+               instrument: Optional[str] = None,
+               oncat: Optional[bool] = True) -> Optional[None]:
+    r"""
+    @brief Test whether the file for a run number exists.
+    @details Search first the datasearch directories and if file is not found, use the locations
+    provided by ONCat
+    @param run_number : just the bare run number, e.g. 12345
```

Comment:
I don't think it's confusing since the documentation clearly states that the input should be a run number, not a nexus file name. What I'll do is insert a check when `run_number` is a string. The check will assert casting the string to an integer.